typings/index.d.ts

```diff
@@ -356,13 +359,6 @@ export function token(value?: string): string;
  */
 export function collection<T>(value: string, addMethods?: CollectionMethods<T>): Collection<T>;
 
-/**
- * Create a temporary Firestorm collection with no methods
- * @param table - The table name to get
- * @returns The collection
- */
-export function table<T>(table: string): Promise<Collection<T>>;
-
```

Comment:
or you could set the method as deprecated

Reply:
smart, will do